lib/FCM/Admin/System.pm

```diff
@@ -130,50 +130,36 @@ sub add_trac_environment {
             $project->get_trac_live_path(),
         ));
     }
-    my @repository_arguments = (q{}, q{});
-    if (-d $project->get_svn_live_path()) {
-        @repository_arguments = (q{svn}, $project->get_svn_live_path());
-    }
     my $RUN = sub{$RUNNER->run(@_)};
     my $TRAC_ADMIN = sub {
         my ($log, @args) = @_;
         my @command = (q{trac-admin}, $project->get_trac_live_path(), @args);
         $RUN->($log, sub {!system(@command)});
     };
-    my $TRAC_ADMIN_CONT = sub {
-        my ($log, @args) = @_;
-        my @command = (q{trac-admin}, $project->get_trac_live_path(), @args);
-        $RUNNER->run_continue($log, sub {!system(@command)});
-    };
     $TRAC_ADMIN->(
         "initialising Trac environment",
         q{initenv},
         $project_name,
         q{sqlite:db/trac.db},
-        @repository_arguments,
         q{--inherit=../../trac.ini},
     );
     my $group = $CONFIG->get_trac_group();
     if ($group) {
         _chgrp_and_chmod($project->get_trac_live_path(), $group);
     }
-    # Note: For some reason, the commands to remove example components,
-    # versions, milestones, priorities fail using the "pip install trac" version
-    # on Travis CI. It is safe to allow the logic to continue after a failure
-    # here as they are really unimportant and can easily be configured later.
     for my $item (qw{component1 component2}) {
-        $TRAC_ADMIN_CONT->(
-            "removing example component $item", q{component remove}, $item,
+        $TRAC_ADMIN->(
+            "removing example component $item", qw{component remove}, $item,
```

Comment:
Fix: quote words instead of quote. The previous behaviour is equivalent to `'component remove'`. This behaviour means that we have a sub-list `..., 'component', 'remove', ...`.

So, removing the above comment on Travis CI as well. It has actually caught this issue correctly. We were ignoring it, and I guess old versions of Trac did not use to care?